pytorch_lightning/trainer/properties.py

```diff
@@ -62,13 +68,18 @@ class TrainerProperties(ABC):
     _weights_save_path: str
 
     accelerator_connector: AcceleratorConnector
+    accumulate_grad_batches: int
```

Comment:
reviewer: the loops are accessing these attributes, so adding them here with types